CHANGES.txt

```diff
@@ -4,7 +4,13 @@ This file will be added to as part of each release
 
 ----
 
-Version 0.2.3, Tue 19 Dec 2017
+Version 0.2.3-local-jquery-ui, Sun 04 Mar 2018
+===============================
+
+b566d7879b	jquery-ui.min.js made available locally (clover-es)
+
+
+Version 0.2.3-local-jquery-ui, Tue 19 Dec 2017
```

Comment:
Same here, leave this out, the `seed` build system will add it in automatically.